pkg/test/resourcefixture/testdata/basic/monitoring/v1beta1/monitoringdashboard/monitoringdashboardbasic/_http.log

```diff
@@ -160,7 +160,7 @@ X-Xss-Protection: 0
             "xyChart": {
               "dataSets": [
                 {
-                  "plotType": "STACKED_BAR",
+                  "plotType": "LINE",
```

Comment:
I had to change these values as part of cross-validating with GCP (which is also why we have some mockgcp changes)